public/framework/catalog_cache/cache_loader.tsx

```diff
@@ -267,7 +267,20 @@ export const useLoadToCache = (loadCacheType: LoadCacheType) => {
     );
   };
 
-  const startLoading = (dataSourceName: string, databaseName?: string, tableName?: string) => {
+interface StartLoadingParams {
+  dataSourceName: string;
+  dataSourceId?: string;
+  databaseName?: string;
+  tableName?: string;
+}
+
+const startLoading = ({
+  dataSourceName,
+  dataSourceId,
+  databaseName,
+  tableName
+}: StartLoadingParams) => {    
+    console.log(dataSourceName,dataSourceId,databaseName,tableName)
```

Comment:
We should remove console.log

Reply:
Also, Is dataSourceId same as `clientId` and `dataSourceClientId`? We should use the same terminology everywhere.